index.html

```diff
@@ -59,21 +60,33 @@ <h2>Pay as You go</h2>
           </article>
         </div>
       </section>
+      <section id="extended">
+
+          <img id="feature" src="/img/feature-img.png" alt="">
```

Comment:
Ideally use vector images like .svg (which you have elsewhere) rather than raster images - more information here https://www.shutterstock.com/blog/raster-vs-vector-file-formats